api/SpeechGrammar.json

```diff
@@ -58,6 +58,162 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "SpeechGrammar": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechGrammar",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "25",
+              "notes": "Speech recognition interfaces are currently prefixed in Chrome, so you'll need to prefix interface names appropriately, e.g. webkitSpeechRecognition; You'll also need to serve your code through a web server for recognition to work. Speech synthesis is fully supported without prefixes.",
```

Comment:
Are these notes redundant? Seems like they could be trimmed down at the very least, since we've already got the flag data in the structure 😄

Reply:
Yes. These notes talk about many things but it's not very concise about SpeechGrammar. I would propose to remove them. Throughout